source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -777,6 +795,9 @@ void Graph::updateMaterials(mx::InputPtr input, mx::ValuePtr value)
             {
                 name = interfaceInput->getNamePath();
             }
+            // Note that if there is a topogical change due to
+            // this value change or a transparency change, then
+            // this is not currently caught here.
             _renderer->getMaterials()[num]->modifyUniform(name, value);
```

Comment:
Leaving a note here as value updates can require new shader generation, or additional inspection like for transparency. First has the same problem in MaterialXView, second works there but not here.